spec/models/case_spec.rb

```diff
@@ -36,70 +36,52 @@
 end
 
 describe 'versioning', versioning: true do
-  it 'starts versioning when a new case is created' do
-    this_case = FactoryBot.create(:case)
-    expect(this_case.versions.size).to eq 1
-    expect(this_case.versions.map(&:event)).to eq %w[create]
-  end
 
   it 'adds a version when the title is changed' do
     this_case = FactoryBot.create(:case)
-    this_case.update_attribute(:title, 'A New Title')
-    expect(this_case.versions.size).to eq 2
-    expect(this_case.versions.map(&:event)).to eq %w[create update]
+    old_title = this_case.title
+    this_case.update!(title: 'A New Title')
+    expect(this_case).to have_a_version_with title: old_title
```

Comment:
This expectation is more "This has a version with the old title" than "a version has been added". I think that changing the it on line 40 or the expectation on 44 so that they match would be better.